File: Maths_And_Stats/Algebra/Factorial/factorial.cs
using System;  
  public class FactorialExample  
   {  
     public static void Main(string[] args)  
      {  
       int i,fact=1,number;      
       Console.Write("Enter a non-negative Number: ");      
       number= int.Parse(Console.ReadLine());     
       for(i=1;i<=number;i++){      
        fact=fact*i;      
       }      
       Console.Write("Factorial of " +number+" is: "+fact);    
     }  
  }  

File: Maths_And_Stats/Algebra/Factorial/factorial.js
let number = parseInt(prompt("Enter a positive integer: "));

if (number < 0) {
  console.log("Factorial for negative number does not exist.");
}
else if (number === 0) {
  console.log(`The factorial of ${number} is 1.`);
}
else {
  let fact = 1;
  for (let i = 1; i <= number; i++) {
    fact *= i;
  }
  console.log(`The factorial of ${number} is ${fact}.`);
}
